packages/client/src/Call.ts

```diff
@@ -1688,4 +1701,93 @@ export class Call {
       { custom: payload },
     );
   };
+
+  registerVideoElement = (
```

Comment:
I'm thinking to have such methods live in `CallWebAPI` class and accessing those would look like:

```ts
call.web.registerVideoElement(...);
```

But I guess that's for the future updates when we have more than one method related to web-only APIs.